### ISSUE TITLE
condVars analysis does not invalidate globals, if special (unknown/library) function is called

### ISSUE THREAD

**Issue by FelixKrayer** [goblint/analyzer#938]
The `combine` transfer function of the condVars analysis just propagates the `ctx.local`. The `ctx.local` could potentially contain information about globals, which might be changed by the library function or unknown function that is called at this point.

I propose to implement some invalidation depending on the library descriptor of the called function.

**Comment by sim642** [comment]
The condvars analysis is in an unknown state overall. See #743.

**Comment by michael-schwarz** [comment]
@sim642 True, but I think it is still relevant to collect specific issues with it, should we chose to fix it later.
